pkg/http/http.go

```diff
@@ -5,7 +5,7 @@ import (
 	"fmt"
 	"net/url"
 
-	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
```

Comment:
ignore this line... but look at line 40 `client := NewClient()`   Now that we import `client` L40 var name of client collides with import package... life would be just wonderful if it didn't :)